gulpfile.js

```diff
@@ -137,6 +139,16 @@ gulp.task("watch", function () {
   gulp.watch(dirs.img + "/**/*.*", ["images"]);
 });
 
+gulp.task("war", function () {
```

Comment:
Sure, this is currently handled in the npm `war` target but maybe it makes sense to have it here as well.